addon/webextension/selector/ui.js

```diff
@@ -541,6 +553,13 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
       return false;
     },
 
+    isElementInViewport(el) {
+      let rect = el.getBoundingClientRect();
+      if (rect.right <= (window.innerWidth || document.documentElement.clientWidth)) {
```

Comment:
Found a reference for clientWidth: https://www.w3.org/TR/cssom-view-1/#dom-element-clientwidth
I think `document.documentElement.clientWidth` is specefically only used for IE8 and earlier versions because they don't support `window.innerWidth`, so maybe we could remove it entirely?
Ref: https://www.w3schools.com/jsref/prop_win_innerheight.asp